SriLanka/UniversityofMoratuwa/faculties_departments.json

```diff
@@ -0,0 +1,92 @@
+{
+    "university_name" : "University of Moratuwa",
+    "faculties" : [
+         {
+            "faculty_name" : "Faculty of Engineering",
+            "web_url" : "http://www.mrt.ac.lk/efac/",
+            "dean_info" : {
+                "name" : "Professor K Kapila CK Perera"
+            },
+            "departments" : [
+                {
+                    "department_name" : "Department of Chemical & Process Engineering",
+                    "web_url" : "http://www.cpe.mrt.ac.lk/",
+                    "hod_info" : {
+                        "name" : "Dr. Gunawardena S. H. P."
+                    }
+                },
+                {
+                    "department_name" : "Department of civil Engineering",
+                    "web_url" : "http://www.civil.mrt.ac.lk/",
+                    "hod_info" : {
+                        "name" : "Bandara J.M.S.J",
+                        "email" : "bandara@civil.mrt.ac.lk",
+                        "contact_no" : "+94-11-2650567 ( Ext : 2129 )"
+                    }
+                },
+                {
+                    "department_name" : "Department of Computer Science & Engineering",
+                    "web_url" : "http://www.cse.mrt.ac.lk/",
+                    "hod_info" : {
+                        "name" : "DR. A. SHEHAN PERERA",
```

Comment:
Can you please change this to Camel case

Reply:
Fixed